### PR TITLE
[1.13.x-blue][RHPAM-4560] download maven 3.8.1 from brew instead of upstream (#1433)

### PR DESCRIPTION
**Pull request by rgdoliveira**
Jira: https://issues.redhat.com/browse/RHPAM-4560

Related PRs:
- https://github.com/kiegroup/kogito-images/pull/1433

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

- <b>(Re)run Jenkins tests</b>  
  Please add comment: <b>Jenkins [test|retest] this</b>

- <b>Prod tests</b>  
  Please add comment: <b>Jenkins (re)run [prod|Prod|PROD]</b>
</details>

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

**Comment by kie-ci1** [comment]
**PR job** `#35` was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images/35/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Prepare offline kogito-examples)<br/>Stage "Prepare offline kogito-examples" skipped due to earlier failure(s)<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Build & Test Images)<br/>Stage "Build & Test Images" skipped due to earlier failure(s)<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/workspace/KIE/kogito/1.13.x-blue/pullrequest/kogito-images/workspaces<br/>[Pipeline] sh<br/>++ docker ps -a -q<br/>+ docker rm -f<br/>"docker rm" requires at least 1 argument.<br/>See 'docker rm --help'.<br/><br/>Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]<br/><br/>Remove one or more containers<br/>+ date<br/>Tue Feb 14 03:06:18 EST 2023<br/>[Pipeline] sh<br/>++ docker images -q<br/>+ docker rmi -f<br/>"docker rmi" requires at least 1 argument.<br/>See 'docker rmi --help'.<br/><br/>Usage:  docker rmi [OPTIONS] IMAGE [IMAGE...]<br/><br/>Remove one or more images<br/>+ date<br/>Tue Feb 14 03:06:19 EST 2023<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/.cekit/cache<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images/35/api/json<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Review by lampajr — APPROVED** [review]

**Comment by kie-ci1** [comment]
**PR job** `#36` was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images/36/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Prepare offline kogito-examples)<br/>Stage "Prepare offline kogito-examples" skipped due to earlier failure(s)<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Build & Test Images)<br/>Stage "Build & Test Images" skipped due to earlier failure(s)<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/workspace/KIE/kogito/1.13.x-blue/pullrequest/kogito-images/workspaces<br/>[Pipeline] sh<br/>++ docker ps -a -q<br/>+ docker rm -f<br/>"docker rm" requires at least 1 argument.<br/>See 'docker rm --help'.<br/><br/>Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]<br/><br/>Remove one or more containers<br/>+ date<br/>Tue Feb 14 04:32:10 EST 2023<br/>[Pipeline] sh<br/>++ docker images -q<br/>+ docker rmi -f<br/>"docker rmi" requires at least 1 argument.<br/>See 'docker rmi --help'.<br/><br/>Usage:  docker rmi [OPTIONS] IMAGE [IMAGE...]<br/><br/>Remove one or more images<br/>+ date<br/>Tue Feb 14 04:32:11 EST 2023<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/.cekit/cache<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images/36/api/json<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

File: modules/kogito-maven/3.8.x/module.yaml [hunk 1]
```diff
@@ -68,8 +68,8 @@ envs:
 # unfortunately by now the version needs to be hardcoded.
 artifacts:
   - name: apache-maven-3.8.1-bin.tar.gz
-    url: https://archive.apache.org/dist/maven/maven-3/3.8.1/binaries/apache-maven-3.8.1-bin.tar.gz
-    md5: 90197f18ed9036a1709ff0e09558c083
+    url: http://download.devel.redhat.com/brewroot/packages/org.apache.maven-apache-maven/3.8.1/1/maven/org/apache/maven/apache-maven/3.8.1/apache-maven-3.8.1-bin.tar.gz
+    sha256: b98a1905eb554d07427b2e5509ff09bd53e2f1dd7a0afa38384968b113abef02
```
Comment (radtriste) [inline modules/kogito-maven/3.8.x/module.yaml:72]: @rgdoliveira looks like `sha256` is not recognized ...
Please also correct on 1.13.x branch ...

**Review by radtriste — CHANGES_REQUESTED** [review]

**Review by spolti — APPROVED** [review]

**Comment by rgdoliveira** [comment]
Closing this PR as it was decided to use rpm for Maven instead of downloading the zip

**Comment by rgdoliveira** [comment]
@radtriste @spolti  I've opened PR https://github.com/kiegroup/kogito-images/pull/1440 adding Tristan's suggestion to fix the PR check (as this PR was closed)

**Pull request closed without merging**